README.md

```diff
@@ -0,0 +1,78 @@
+# secret-script
+Proof-of-concept python script for sychronization between kubernetes secret and gcloud secret manager.
+
+## Prerequisites
+- Create a gke cluster.
+
+- [Create service account for app](https://cloud.google.com/docs/authentication/production#command-line)
+
+- Grant required permissions to the service account `service-account-name`.
```

Comment:
You'll need `roles/secretmanager.admin` to work with Secret Manager secrets

Reply:
missed that, added.